CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Dangerzone 0.4.1 (unreleased)
+
+- Platform support: Drop Fedora 35, since it's end-of-life ([issue #308](https://github.com/freedomofpress/dangerzone/issues/308))
+- Feature: Introduce PySide6 / Qt6 support on Windows, MacOS, and Linux (dev-only) ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Feature: Adjust conversion timeouts based on the document's pages/size, and
+  allow users to disable them ([issue #327](https://github.com/freedomofpress/dangerzone/issues/327))
+- Feature: Convert documents with non-standard permissions or SELinux labels ([issue #335](https://github.com/freedomofpress/dangerzone/issues/335))
+- Feature: Add version info in the CLI and GUI ([issue #219](https://github.com/freedomofpress/dangerzone/issues/219))
+- Bug fix: Remove unused PDFtk and sudo libraries from the container image, to
+  lower its attack surface and reduce its size ([issue #232](https://github.com/freedomofpress/dangerzone/issues/232))
+- Bug fix: Report exceptions during conversions ([issue #309](https://github.com/freedomofpress/dangerzone/issues/309))
+- Bug fix: (Windows) Fix Dangerzone description on "Open With" ([issue #283](https://github.com/freedomofpress/dangerzone/issues/283))
+- Bug fix: Update Linux instructions for development on Qubes
+- Bug fix: Do not print debug logs in end-user executables
```

Comment:
We probably also want to add the respective issues here. I see that there was no parent one, but maybe we could make it [issue #316](https://github.com/freedomofpress/dangerzone/issues/316). It's kind of related and was also closed. If not, we could mention the PR that fixed this: https://github.com/freedomofpress/dangerzone/pull/301

Reply:
Nice catch, I can refer to that issue.